indico/modules/events/persons/templates/management/_person_list_row.html

```diff
@@ -0,0 +1,62 @@
+{% macro render_person_row(person_data) %}
+    {% set person = person_data.person %}
+    <td class="i-table checkbox-column">
+        <input type="checkbox" value="{{ person.id }}" class="select-row" name="person_id"
+               {{ 'disabled' if not person.email }}>
+    </td>
+    <td class="i-table name-column">{{ person.get_full_name(last_name_first=true, last_name_upper=false,
+                                                abbrev_first_name=false) }}</td>
+    <td class="i-contributiontable email-column">{{ person.email }}</td>
+    <td class="i-table affiliation-column">{{ person.affiliation }}</td>
+    <td class="i-table">
+        <span class="i-label{{ ' no-role' if not person_data.roles.chairperson }}">
```

Comment:
:+1: I can imagine this was added with the goal of not having trailing whitespace in the attribute value, but I don't think it makes much sense caring about purity of compiled HTML.